cinema/views.py

```diff
@@ -56,3 +91,12 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderListViewSet(viewsets.ModelViewSet):
```

Comment:
![image](https://github.com/mate-academy/py-tickets-orders/assets/103201010/5b2290c3-64ea-47bf-a5a1-ddad91d5d05a)
fix this 
also are u sure about naming of a viewset ? what do u mean with LIST view set ?